<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NuevoDialog</class>
 <widget class="QDialog" name="NuevoDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>383</width>
    <height>397</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblGrupo">
       <property name="text">
        <string>Grupo o compañía minera:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="cboLista"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QListWidget" name="lwUnidades"/>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="lblNombre">
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="txtNombre"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="lblDescripcion">
       <property name="text">
        <string>Descripción:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPlainTextEdit" name="teDescripcion"/>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnGuardar">
       <property name="toolTip">
        <string>(Ctrl+S)</string>
       </property>
       <property name="text">
        <string>Guardar</string>
       </property>
       <property name="icon">
        <iconset resource="rsc.qrc">
         <normaloff>:/img/save.png</normaloff>:/img/save.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+S</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancelar">
       <property name="toolTip">
        <string>(Esc)</string>
       </property>
       <property name="text">
        <string>Cancelar</string>
       </property>
       <property name="icon">
        <iconset resource="rsc.qrc">
         <normaloff>:/img/cancel.png</normaloff>:/img/cancel.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Esc</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
